networkx/algorithms/centrality/tests/test_load_centrality.py

```diff
@@ -41,6 +44,27 @@ def test_not_strongly_connected(self):
             assert result[n] == pytest.approx(b[n], abs=1e-3)
             assert result[n] == pytest.approx(nx.load_centrality(self.D, n), abs=1e-3)
 
+    def test_P2_normalized_load(self):
+        G = self.P2
+        c = nx.load_centrality(G, normalized=True)
+        d = {0: 0.000, 1: 0.000}
+        for n in sorted(G):
+            assert c[n] == pytest.approx(d[n], abs=1e-3)
+
+    def test_C2_normalized_load(self):
+        G = self.C2
```

Comment:
This all looks good :)
But we don't really need to add `test_C2_normalized_load` and `test_K2_normalized_load`. Just one test is enough :)

Reply:
So could you remove them and just keep `test_P2_normalized_load`?

Thanks!

Reply:
Okay! will solve this right now. Thanks for your guidance.